hydroflow_lang/src/graph/flat_to_partitioned.rs

```diff
@@ -1,4 +1,4 @@
-use std::collections::{HashMap, HashSet};
+use std::collections::{BTreeMap, BTreeSet};
```

Comment:
I wonder if we could do this in `cfg(test)` and use regular `HashMap` otherwise?